### ISSUE TITLE
bin/omero-importer

### ISSUE THREAD

**Issue by jburel** [ome/omero-insight#25]
bin/omero-importer does not work
```Could not find or load main class org.openmicroscopy.shoola.Main```

**Comment by mtbc** [comment]
Still true for 5.5.8.

**Comment by PaulVanSchayck** [comment]
Still true for 5.5.9.

In case the cause was unclear. The startup scripts misses the contents of the CLASSPATH variable. After unzipping. Look in file `bin/omero-importer` file. The `CLASSPATH=` is empty.

When starting with empty `CLASSPATH`, you get the following error
```
$ ./bin/omero-importer 
Error: Could not find or load main class org.openmicroscopy.shoola.Main
```

Copying over the contents of the variable from the insight startup script fixes the issue.

**Comment by jburel** [comment]
@PaulVanSchayck are you the importer? The reason I am asking is that we are considering not building the artifacts anymore

**Comment by PaulVanSchayck** [comment]
Haha.. you've been considering that for years, haven't you? Every time I report an issue with the importer I get told exactly this :P.

We like the simple GUI. Especially because we tell users to use the webclient mostly, and hardly anyone uses the insight client. So we have only the importer installed on our microscopes and workstations.

**Comment by jburel** [comment]
See https://forum.image.sc/t/omero-desktop-client-omero-insight/35234
The Desktop client will stay e.g. insight.dmg, we were just considering not bundling the importer as a standalone app.
If it is useful, we will keep bundling it.

**Comment by PaulVanSchayck** [comment]
Hmm I see..

Would it be an option to include the `omero-importer` startup script in the insight artifact? Or as command line option to the `insight` startup script? Or do you think that's confusing?

**Comment by jburel** [comment]
I have fixed the problem.
We did not hear that people still needed and it seems that you still do so we will keep building the artifact
I have also automated the build via travis so it should not be a problem.

**Comment by jburel** [comment]
see https://github.com/ome/omero-insight/pull/127

**Comment by jburel** [comment]
Fixed in 5.5.10
